contrib/to-a11y-string/to-a11y-string.js

```diff
@@ -0,0 +1,735 @@
+                case "inner": {
+                    buildA11yStrings(tree, a11yStrings, atomType);
+                    break;
+                }
+                case "open": {
+                    buildString(text, "open", a11yStrings);
+                    break;
+                }
+                case "punct": {
+                    buildString(text, "punct", a11yStrings);
+                    break;
+                }
+                case "rel": {
+                    buildString(text, "rel", a11yStrings);
+                    break;
+                }
+                default: {
+                    (tree.family: empty);
+                    throw new Error(`"${tree.family}" is not a valid atom type`);
+                }
+            }
+            break;
+        }
+
+        case "color": {
+            const color = tree.color.replace(/katex-/, "");
+
+            buildRegion(a11yStrings, (regionStrings) => {
+                regionStrings.push("start color " + color);
+                buildA11yStrings(tree.body, regionStrings, atomType);
+                regionStrings.push("end color " + color);
+            });
+            break;
+        }
+
+        case "color-token": {
+            throw new Error("TODO: color-token");
+        }
+
+        case "delimsizing": {
+            if (tree.delim && tree.delim !== ".") {
+                buildString(tree.delim, "normal", a11yStrings);
+            }
+            break;
+        }
+
+        case "genfrac": {
+            buildRegion(a11yStrings, (regionStrings) => {
+                // genfrac can have unbalanced delimiters
+                const {leftDelim, rightDelim} = tree;
+
+                // NOTE: Not sure if this is a safe assumption
+                // hasBarLine true -> fraction, false -> binomial
+                if (tree.hasBarLine) {
+                    regionStrings.push("start fraction");
+                    leftDelim && buildString(leftDelim, "open", regionStrings);
+                    buildA11yStrings(tree.numer, regionStrings, atomType);
+                    regionStrings.push("divided by");
+                    buildA11yStrings(tree.denom, regionStrings, atomType);
+                    rightDelim && buildString(rightDelim, "close", regionStrings);
+                    regionStrings.push("end fraction");
+                } else {
+                    regionStrings.push("start binomial");
+                    leftDelim && buildString(leftDelim, "open", regionStrings);
+                    buildA11yStrings(tree.numer, regionStrings, atomType);
+                    regionStrings.push("over");
+                    buildA11yStrings(tree.denom, regionStrings, atomType);
+                    rightDelim && buildString(rightDelim, "close", regionStrings);
+                    regionStrings.push("end binomial");
+                }
+            });
+            break;
+        }
+
+        // katex: function(tree: ParseNode<"katex">, a11yStrings) {
```

Comment:
Comment block can be deleted?

Reply:
It can.  I'll delete.  I don't we use `\katex` in any of our content anyways.